store/tikv/mock-tikv/cop_handler_dag.go

```diff
@@ -65,7 +65,11 @@ func (h *rpcHandler) handleCopDAGRequest(req *coprocessor.Request) (*coprocessor
 	}
 	var chunks []tipb.Chunk
 	for {
-		handle, row, err := e.Next()
+		var (
+			handle int64
+			row    [][]byte
+		)
+		handle, row, err = e.Next()
```

Comment:
This pull request wants to expose all possible shadow mistakes. So it is worth to pass all the shadowed variables down to test the developer's purposes, e.g. #3910